diff --git a/board/qemu/riscv32-vexriscv/linux.config.fragment b/board/qemu/riscv32-vexriscv/linux.config.fragment
index 9156d1e72d..5835a03bf5 100644
--- a/board/qemu/riscv32-vexriscv/linux.config.fragment
+++ b/board/qemu/riscv32-vexriscv/linux.config.fragment
@@ -1,22 +1,5 @@
 CONFIG_BLK_DEV_INITRD=y
 CONFIG_DEVTMPFS=y
 CONFIG_DEVTMPFS_MOUNT=y
-CONFIG_OF=y
-CONFIG_INPUT_FF_MEMLESS=y
-# CONFIG_HID_A4TECH is not set
-# CONFIG_HID_APPLE is not set
-# CONFIG_HID_BELKIN is not set
-# CONFIG_HID_CHERRY is not set
-# CONFIG_HID_CHICONY is not set
-# CONFIG_HID_CYPRESS is not set
-# CONFIG_HID_EZKEY is not set
-# CONFIG_HID_ITE is not set
-# CONFIG_HID_KENSINGTON is not set
-# CONFIG_HID_LOGITECH is not set
-# CONFIG_HID_REDRAGON is not set
-# CONFIG_HID_MICROSOFT is not set
-# CONFIG_HID_MONTEREY is not set
 CONFIG_TMPFS=y
 CONFIG_TMPFS_POSIX_ACL=y
-CONFIG_FRAME_WARN=1024
-# CONFIG_UNUSED_SYMBOLS is not set
